rebar.config

```diff
@@ -68,6 +68,8 @@
         vmq_server,
         vernemq_dev,
         credentials_obfuscation,
+        wx,
```

Comment:
For a pure server build, we don't need `wx` but we could use multiple build profiles and have a profile for developing.

Reply:
Oh, yeah. That is a left-over from debugging. Adding a profile which supports the debugger would be nice. Once I got it running, I have fallen in love with the gui debugger ;-)

Reply:
it is cool!
to add a profile, we can add sth like the following to the Makefile:
```
dev_build: PROFILE = as develop
dev_build: rel
```

Then add `profiles` to the rebar.config (keeping the main relx build out of it):

```
{profiles, [
   {develop, [
        {relx, [
            {release, {vernemq, semver}, [
                sasl,
                inets,
                xmerl....]}]}]}
                
           ]}.
```
then build with     
`make dev_build`